### ISSUE TITLE
Skip to links not working

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1436]
#1431 is related (Skip to main content should be first)

- The "skip to first result" and "skip to search" are not working.
- Per SODA, the skip to search should go to the search form container, and preferably, the input has a focus state. See example on the screenshot below from this bootstrap page - https://getbootstrap.com/docs/5.1/forms/form-control/. (I'm wondering if these two statements contradict each other... or at the very least, we can do them at the same time... skip to search form and focus on input. If they are, then I'd suggest skip should go to input and focus state should be triggered.) 

<img width="851" alt="Screenshot 2024-10-16 at 4 47 52 PM" src="https://github.com/user-attachments/assets/d63e0bbf-6f2f-465f-9f47-db30bdd1bcab">

**Comment by thatbudakguy** [comment]
Possibly resolved by https://github.com/projectblacklight/blacklight/issues/3302.

**Comment by dnoneill** [comment]
https://github.com/projectblacklight/blacklight/issues/3302 has been merged, should be fixed with new release of blacklight.

**Comment by thatbudakguy** [comment]
needs another blacklight 8.x release so we can pull in the change.

**Comment by dbranchini** [comment]
These links are working, except "Skip to search" lands slightly below the search form so you can't see the input. It's working correctly on Exhibits. I'll file a separate ticket for this issue.
